src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -567,8 +567,8 @@ public void addChild(DLNAResource child, boolean isNew) {
 					}
 
 					// Hide watched videos depending user preference
-					if (configuration.getFullyPlayedAction() == 2 && child.format.isVideo() && MediaMonitor.isWatched(child.getSystemName())) {
```

Comment:
Format is unreliable to determine content type, several (container) formats can either audio or video. That was the idea behind the "new" methods on media instead of format, they actually look at the content not the file extension.